app/assets/javascripts/admin/models/report.js.es6

```diff
@@ -374,14 +374,14 @@ const Report = Discourse.Model.extend({
 
   _secondsLabel(value) {
     return {
-      value,
+      value: typeof value === "number" ? value : Number(value),
```

Comment:
We might want to extract this into an helper method so that we don't have to repeat it 4 times.

Reply:
Also we have this:

https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/lib/utilities.js.es6#L602

Not sure if we should use this, or check on type.